apps/dotcms-ui/src/app/api/services/dot-apps/dot-apps.service.spec.ts

```diff
@@ -239,7 +239,7 @@ describe('DotAppsService', () => {
 
     it('should throw error on Save a specific app and handle it', () => {
         const params = {
-            name: { hidden: 'false', value: 'test' }
+            name: { hidden: false, value: 'test' }
```

Comment:
Are you sure of this? it could be that dotcms response is a string.

Reply:
Those are the arguments we send to the method that makes the request, not the response.